### PR TITLE
fix(deps): Update chalk

### PR DESCRIPTION
**Pull request by massongit**
https://github.com/textlint/textlint/issues/840

`ansi-regex` 2.1.1 ~ 5.0.0 is vulnerable to Inefficient Regular Expression Complexity: CVE-2021-3807
Therefore, I update `chalk`.

https://github.com/chalk/chalk/releases/tag/v2.0.0
>Removed `chalk.stripColor()`. Use the [strip-ansi](https://github.com/chalk/strip-ansi) package directly instead. https://github.com/chalk/chalk/commit/04cae226cc0fc11fd7898f3fa91fdc4a3b3e496b

In updating, I replace `chalk.stripColor` to `stripAnsi`.

### CODE REVIEW

File: packages/@textlint/fixer-formatter/package.json [hunk 1]
```diff
@@ -39,11 +39,11 @@
   "dependencies": {
     "@textlint/module-interop": "^12.1.0",
     "@textlint/types": "^12.1.0",
-    "chalk": "^1.1.3",
+    "chalk": "^4.1.2",
     "debug": "^4.3.3",
     "diff": "^4.0.2",
     "is-file": "^1.0.0",
-    "string-width": "^1.0.2",
+    "string-width": "^4.2.3",
```
Comment (azu) [inline packages/@textlint/fixer-formatter/package.json:46]: Confirm: No API changes.
https://github.com/sindresorhus/string-width/compare/v1.0.2...v2.0.0
https://github.com/sindresorhus/string-width/releases/tag/v3.0.0
https://github.com/sindresorhus/string-width/releases/tag/v4.0.0

Comment (massongit) [reply packages/@textlint/fixer-formatter/package.json]: `string-width` do not have to update.
Therefore, `string-width` is excluded from the update target: https://github.com/textlint/textlint/pull/842/commits/902c362af39f019a0170d2d384a3f3f38e9bbceb

Comment (azu) [reply packages/@textlint/fixer-formatter/package.json]: Will it be separated PR?

I'm ok that a single PR includes multiple package updates.

Comment (massongit) [reply packages/@textlint/fixer-formatter/package.json]: After check again, it seems that it is not necessary to update `string-width` to update `ansi-regex`.
Therefore, it was excluded from the update target.

Comment (azu) [reply packages/@textlint/fixer-formatter/package.json]: OK.
It seems like separated issue.
I've created it as new issue.
https://github.com/textlint/textlint/issues/843

**Pull request merged**